net-p2p/freenet/files/0.7.5_p1491-update-for-jna-5.x.patch

```diff
@@ -0,0 +1,31 @@
+From 830b4bb3948bd69fbd9b10001940f1775051114a Mon Sep 17 00:00:00 2001
```

Comment:
Does this need upstreaming to https://github.com/freenet/fred?

Reply:
The upstream is still using JNA 4.5.2 in the Gradle project. If they don't want to support using different versions of dependencies, then this patch is not quite useful to them. It is upstream-able though

Reply:
Ah, I see. I would still say it's worth sending to them though for future proofing.